File: user-rest-service-1.0.2/generated-code/csharp/src/main/csharp/io/swagger/Model/ShowcaseDatatypeDate.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;



namespace io.swagger.Model {
  public class ShowcaseDatatypeDate {
    

    
    public DateTime Date { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ShowcaseDatatypeDate {\n");
      
      sb.Append("  Date: ").Append(Date).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: user-rest-service-1.0.2/generated-code/jaxrs/src/main/java/io/swagger/model/ShowcaseDatatypeDate.java
package io.swagger.model;

import java.util.Date;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;



/**
 * 
 **/
@ApiModel(description = "")
public class ShowcaseDatatypeDate  {
  
  private Date date = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("date")
  public Date getDate() {
    return date;
  }
  public void setDate(Date date) {
    this.date = date;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ShowcaseDatatypeDate {\n");
    
    sb.append("  date: ").append(date).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}


